app/web/features/communities/discussions/DiscussionPage.test.tsx

```diff
@@ -237,22 +237,25 @@ describe("Discussion page", () => {
         return getThread(threadId);
       });
       renderDiscussion();
-      await waitForElementToBeRemoved(screen.getByRole("progressbar"));
 
-      userEvent.click(
-        screen.getByRole("button", {
+      const user = userEvent.setup();
+
+      user.click(
+        await screen.findByRole("button", {
           name: t("communities:load_earlier_comments"),
         })
       );
-      await waitForElementToBeRemoved(screen.getByRole("progressbar"));
+
```

Comment:
Move the comment where it was next to assertion to here so it's clear why we're asserting the function has been called 6 times?

Reply:
Oh yes good catch!